packages/dds/merge-tree/package.json

```diff
@@ -79,14 +79,14 @@
 	},
 	"devDependencies": {
 		"@fluid-internal/stochastic-test-utils": ">=2.0.0-internal.4.0.0 <2.0.0-internal.5.0.0",
+		"@fluid-internal/test-pairwise-generator": ">=2.0.0-internal.4.0.0 <2.0.0-internal.5.0.0",
```

Comment:
marking `test-pairwise-generator` as `private: true` will cause problems for the DDS fuzz pipeline, as that pipeline aims to avoid rebuilding internal code, so needs to be able to install `@fluid-internal/test-pairwise-generator` from somewhere.

I'm still a fan of moving these packages to the @fluid-internal scope for the benefits of being able to make breaking changes and no longer care about their APIs. The feed we publish @fluid-internal packages to (which is called dev, *not* internal, confusingly enough) is internal and documented as only to be used by our team.

Reply:
@Abe27342 Do you have a pointer to what that pipeline needs to install?  I'm building out a list of changes to hide more packages so that would be very helpful!

Reply:
right now it's any dev dep of merge-tree, sequence, and @fluid-experimental/tree. There have been a few other additions of fuzz tests to other DDS packages that I've been meaning to add. But it's easy enough to un-private something once the packages get added there.